hoomd/pytest/test_simulation.py

```diff
@@ -333,8 +333,9 @@ def test_run_limit(simulation_factory, lattice_snapshot_factory):
         sim.run(-1)
 
 
-def test_seed(simulation_factory, lattice_snapshot_factory):
-    sim = simulation_factory()
```

Comment:
Why this change? `simulation_factory` takes care of some important things like reducing the sorter grid size to run unit tests faster. While the sorter is never activated in this test, this pattern may get copied and pasted elsewhere.

Reply:
Yes this is to prevent developers from not setting a seed when they should and avoid testing error messages.